src/framework/pagination/Pagination.spec.jsx

```diff
@@ -0,0 +1,76 @@
+
+import { TestCaseFactory } from 'react-test-kit';
+import Pagination from './Pagination.jsx';
+
+describe('Pagination', () => {
+  describe('Props', () => {
+    describe('page, pages and maxVisiblePages', () => {
```

Comment:
`currentPage, totalPages and visiblePages`

Reply:
Oups, thanks...